packages/one-app-bundler/package.json

```diff
@@ -48,7 +48,7 @@
     "babel-preset-amex": "^3.2.0",
     "chalk": "^2.4.2",
     "core-js-compat": "^3.4.2",
-    "css-loader": "3.3.0",
+    "css-loader": "^3.4.2",
```

Comment:
we should keep this locked down in case this changes again

Reply:
Fixed in d6df64e